npm-check.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava';
+import npmCheck from "./lib/index.js";
+
+test('should be possible to import and run the library against the current directory', async t => {
+    const currentState = await npmCheck({});
+    const packages = await currentState.get('packages');
+    t.truthy(packages.length > 0);
+});
```

Comment:
Can we add a test that runs the cli tool as well? As that was the previous test

Reply:
On this branch, I've changed the `npm test` script to:

```json
"test": "npm run lint && ava && ./bin/cli.js || echo Exit Status: $?."
```

So I've added the `ava` test to the existing test of running `cli.js`, rather than replacing it.